webapp/Connector/src/utility/Query.js

```diff
@@ -402,6 +483,8 @@ Ext.define('Connector.utility.Query', {
 
         Ext.iterate(datasets, function(name)
         {
+            if (gridOptions && gridOptions.dataSource && datasets[name].queryName != gridOptions.dataSource.toLowerCase() && name != 'cds.gridbase')
```

Comment:
Don't we have a constant for `cds.gridbase` and/or should this be comparing against `[Connector.studyContext.gridBaseSchema, Connector.studyContext.gridBase].join('.')`?